app/views/partials/header.ejs

```diff
@@ -5,6 +5,14 @@
     <meta charset="utf-8">
     <meta data-build-hash="<%=commitHash%>">
     <meta name="viewport" content="width=device-width">
+    <% if(typeof article !== 'undefined') { %>
+        <meta name="description" content="<%=article.description%>"">
```

Comment:
Google advices that meta tag shouldn't be a duplicate of already existing content on the page.

Reply:
Is the description also the first paragraph?

Reply:
Article description is it's own property and content, if they are duplicates that is the fault of the article author.

Reply:
So we don't display description in the article's content?